tornjak-frontend/src/components/agent-create-join-token.tsx

```diff
@@ -130,7 +136,7 @@ class CreateJoinToken extends Component<CreateJoinTokenProp, CreateJoinTokenStat
       const [isIdValid, trustDomain, path] = this.parseSpiffeId(this.state.spiffeId)
       
       if (!isIdValid) {
-        displayError("Invalid SPIFFE id.")
+        showToast({caption: "The spiffe id is invalid."})
```

Comment:
```suggestion
        showToast({caption: "The SPIFFE id is invalid."})
```

This is an acronym, so let's keep it uppercase